### PR TITLE
implement roots.clean()

### PR DESCRIPTION
**Pull request by kylemac**

### CODE REVIEW

**Comment by kylemac** [comment]
snuck in a readme commit in this PR. i think this makes the constructor options a bit more clear

**Comment by jenius** [comment]
@kylemac definitely, looks much better. Thanks!

File: lib/cli/index.js [hunk 1]
```diff
@@ -42,6 +43,7 @@ export default class CLI extends EventEmitter {
     project.on('error', this.emit.bind(this, 'error'))
     project.on('warning', this.emit.bind(this, 'warning'))
     project.on('compile', this.emit.bind(this, 'compile'))
+    project.on('removed', this.emit.bind(this, 'removed'))
```
Comment (jenius) [inline lib/cli/index.js]: maybe `remove`, just to keep tense consistent?

File: lib/cli/index.js [hunk 2]
```diff
@@ -76,4 +78,16 @@ export default class CLI extends EventEmitter {
...
     s.setDefaults({ fn: 'watch' })
   }
+
+  addClean () {
+    const s = this.sub.addParser('clean', { help: 'Removes the output directory of a roots project' })
+
+    s.addArgument(['path'], {
+      nargs: '?',
+      defaultValue: process.cwd(),
+      help: 'Path to a project that you would like to compile'
```
Comment (jenius) [inline lib/cli/index.js]: path to a project you would like to _remove_?

**Comment by kylemac** [comment]
@jenius addressed your review feedback

**Comment by jenius** [comment]
:+1: lgtm, squash and merge!

**Pull request merged**